tests/helper.py

```diff
@@ -1,9 +1,21 @@
 from errbot.backends.test import TestBot
+from errbot.plugin_info import PluginInfo
+from errbot.templating import add_plugin_templates_path
+from pathlib import Path
 
 
 def plugin_testbot(klass, loglevel, config=None):
```

Comment:
Do we even need this after `CoroboTestCase`?

Reply:
Nope